### PR TITLE
feat: add project should fetch from config.json

### LINKED ISSUE #723: Add project should fetch config.json
We want to deprecate `env.json`.

### PR DESCRIPTION
**Pull request by asthabh23**
fix #723

### CODE REVIEW

File: src/extension/utils.js [hunk 1]
```diff
@@ -566,8 +566,8 @@ export async function getProjectEnv({
     if (project) {
       env.project = project;
     }
-    if (contentSourceUrl) {
-      env.mountpoints = [contentSourceUrl];
+    if (mountpoints) {
+      env.mountpoints = mountpoints;
```
Comment (rofe) [inline src/extension/utils.js:570]: Keep it as an array

**Review by rofe — CHANGES_REQUESTED** [review]

File: test/extension.test.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ describe.skip('Test sidekick extension', () => {
             prod: {
               host: 'www.example.com',
             },
-            contentSourceUrl: 'https://example.sharepoint.com/documents/website',
+            mountpoints: 'https://example.sharepoint.com/documents/website',
```
Comment (rofe) [inline test/extension.test.js:64]: This should be an array

File: test/mocks/fetchMock.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const HELIX_ENV_JSON = {
     host: 'live.example.com',
   },
   project: 'Adobe Business Website',
-  contentSourceUrl: 'https://adobe.sharepoint.com/:f:/s/Dummy/Alk9MSH25LpBuUWA_N6DOL8BuI6Vrdyrr87gne56dz3QeQ',
+  mountpoints: 'https://adobe.sharepoint.com/:f:/s/Dummy/Alk9MSH25LpBuUWA_N6DOL8BuI6Vrdyrr87gne56dz3QeQ',
```
Comment (rofe) [inline test/mocks/fetchMock.js:28]: Delete `HELIX_ENV_JSON` and add a mock for a `CONFIG_JSON` instead

File: test/extension.test.js (hunk 2 above)
Comment (asthabh23) [reply test/extension.test.js]: fixed in [e5b214b](https://github.com/adobe/helix-sidekick-extension/pull/727/commits/e5b214b202d1e462982b19e00ef5de3a2696d022)

File: src/extension/utils.js (hunk 1 above)
Comment (asthabh23) [reply src/extension/utils.js]: fixed in [e5b214b](https://github.com/adobe/helix-sidekick-extension/pull/727/commits/e5b214b202d1e462982b19e00ef5de3a2696d022)

File: test/mocks/fetchMock.js (hunk 3 above)
Comment (asthabh23) [reply test/mocks/fetchMock.js]: fixed in [e5b214b](https://github.com/adobe/helix-sidekick-extension/pull/727/commits/e5b214b202d1e462982b19e00ef5de3a2696d022)

File: src/extension/utils.js [hunk 4]
```diff
@@ -542,32 +542,32 @@ export async function getProjectEnv({
...
+    if (host) {
+      env.host = host;
     }
     if (project) {
       env.project = project;
     }
-    if (contentSourceUrl) {
-      env.mountpoints = [contentSourceUrl];
+    if (mountpoints) {
+      env.mountpoints = Array.isArray(mountpoints) ? mountpoints : [mountpoints];
```
Comment (rofe) [inline src/extension/utils.js:570]: Can it ever not be an array?

**Review by rofe — APPROVED** [review]

**Review by rofe — APPROVED** [review]

File: src/extension/utils.js (hunk 4 above)
Comment (asthabh23) [reply src/extension/utils.js]: As discussed, I have removed the explicit check for array type in [1b6af8c](https://github.com/adobe/helix-sidekick-extension/pull/727/commits/1b6af8c69c6e2fee15ed79f5a97d456f5b96c23c)(trusting that admin service will always return an array)

**Pull request merged**